browser-test/src/application_navigation.test.ts

```diff
@@ -896,6 +896,52 @@ describe('Applicant navigation flow', () => {
       await applicantQuestions.expectIneligiblePage()
     })
 
+    it('ineligible page renders markdown', async () => {
+      const {
+        page,
+        adminQuestions,
+        applicantQuestions,
+        adminPredicates,
+        adminPrograms,
+      } = ctx
+      const questionName = 'question-with-markdown'
+      const programName = 'Program with markdown question'
+
+      // Add a question with markdown in the question text
+      await loginAsAdmin(page)
+      await adminQuestions.addTextQuestion({
+        questionName: questionName,
+        questionText:
+          'This is a _question_ with some [markdown](https://www.example.com)',
+      })
+      await adminPrograms.addProgram(programName)
+      await adminPrograms.editProgramBlock(programName, 'first description', [
+        questionName,
+      ])
+      // Add an eligiblity condition on the markdown question
+      await adminPrograms.goToEditBlockEligibilityPredicatePage(
+        programName,
+        'Screen 1',
+      )
+      await adminPredicates.addPredicate(
+        questionName,
+        /* action= */ null,
+        'text',
+        'is equal to',
+        'foo',
+      )
+      await adminPrograms.gotoAdminProgramsPage()
+      await adminPrograms.publishProgram(programName)
+      await logout(page)
+
+      // Apply to the program and answer the eligibility question with an ineligible answer
+      await applicantQuestions.applyProgram(programName)
+      await applicantQuestions.answerTextQuestion('bar')
+      await applicantQuestions.clickNext()
+      await applicantQuestions.expectIneligiblePage()
+      await validateScreenshot(page, 'ineligible-page-with-markdown')
```

Comment:
nit: I just added mobile screenshots for the other tests in this file. If you change it to this, it will take a second screenshot on mobile

await validateScreenshot(page, 'ineligible-page-with-markdown', /* fullPage= */ true, /* mobileScreenshot= */ true)

Reply:
that's awesome! thanks for adding that. i updated this call to generate the mobile screenshot.